requirements.txt

```diff
@@ -0,0 +1,2 @@
+numpy
+requests
```

Comment:
Should requests be included? Or is it "optional"? It's only used in `utils.py:fetch_mnist`.

Reply:
Stream says: "from `numpy` scratch", so do we need `requests`? 😉

Reply:
Well it is in https://github.com/geohot/tinygrad/blob/5216a1d9f3c4848e20cc1c8db3db77a4721c2bca/tinygrad/utils.py#L14

Since [we're all devs](https://github.com/geohot/tinygrad/pull/14#issuecomment-714870069), I'm going to make a single reqs.txt and include it. Logic being "reqs.txt is there so that developers can get up and running quickly" not "reqs.txt is there to help distribute this as a package"